src/sparseml/sparsification/modifier_pruning.py

```diff
@@ -27,7 +27,12 @@
     ModifierProp,
 )
 from sparseml.sparsification.types import SparsificationTypes
-from sparseml.utils import ALL_TOKEN, convert_to_bool, validate_str_iterable
+from sparseml.utils import (
+    ALL_TOKEN,
+    FROM_PARAM_TOKEN,
```

Comment:
What's the function of `FROM PARAM TOKEN`?

Reply:
this is a keyword for `init_sparsity` that will initialize sparsity levels in a pruning modifier to the existing sparsity in the model